src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -18,6 +26,21 @@
 (def ^:private opacity-from-default 0.5)
 (def ^:private signal-anim-duration 3900)
 (def ^:private signal-anim-duration-2 1950)
+(def ^:private min-audio-duration-ms 500)
+(def ^:private max-audio-duration-ms 120000)
+(def ^:private metering-interval 100)
+(def ^:private base-filename "am.")
+(def ^:private default-format "aac")
+
+(def ^:private record-audio-worklets (js/require "../src/js/record_audio_worklets.js"))
+(def ^:private thumb-light (js/require "../resources/images/icons2/12x12/thumb-light.png"))
+(def ^:private thumb-dark (js/require "../resources/images/icons2/12x12/thumb-dark.png"))
+
```

Comment:
i remember there's discussion about using `status-im.react-native.resources` instead of `js/require`
oh found it https://github.com/status-im/status-mobile/pull/14710#discussion_r1066521632

of course we can do this in one go as suggested in the comment

Reply:
FYI: I just created this issue to help us solve this tech debt https://github.com/status-im/status-mobile/issues/14801